src/main/webapp/index.html

```diff
@@ -8,7 +8,42 @@
 
 <div id="navbar-container"></div>
 <div class="container">
-  <h1 class="display-2 mb-5 mt-5">The Open Source Community is for <span class="emphasis">Everyone</span></h1>
+  <h1 class="display-2 mb-5 mt-5 text-center">The Open Source Community is for <span class="emphasis">Everyone</span></h1>
+  <!-- TODO: Change button text/link depending on user authorization. -->
+  <div class="d-flex justify-content-center"><div class="btn btn-emphasis mt-5 mb-5 text-center btn-lg btn-block btn-main">Create a Profile!</div></div>
+</div>
+<div class="emphasis-box mt-5 mb-5">
```

Comment:
nit: Could we add a small vertical padding to the "About Us" div?

[With padding](https://screenshot.googleplex.com/hCodj3JKLxu)
[Without padding](https://screenshot.googleplex.com/1BCkedMegGQ)

```suggestion
<div class="emphasis-box mt-5 mb-5 py-1">
```